Week 4/gemCollector/src/InputManager.java

```diff
@@ -0,0 +1,132 @@
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+// Analyze the input file and save the information in useful format.
+public class InputManager {
+  private PointInfo[][] pointInfoMap;
+  private int mapSizeY = -1;
+  private int mapSizeX = -1;
+  private Point startPoint = new Point();
+  private Point endPoint = new Point();
+
+  // For a given file, analyze the file.
+  public void getInput(String filename) throws Exception {
```

Comment:
I said please write a function for a module. You write every module in this function